### DISCUSSION TITLE
Trying to install osiset on laravel 9

### DISCUSSION THREAD

**Discussion by rizwanWixpa098** [osiset/laravel-shopify#1152] [General]
Hi,
I'm trying to install osiset package on laravel 9 app. But I think laravel 9 is not supported by osiset package, correct me if I'm wrong. Is there anyone else who encountered this issue??

**Comment by zeeshanhaider6236dev** [comment]
i'm also facing this kind of issue. my client want me to develop shopify app on laravel 9 but i'm getting error in osiset package installation

**Comment by zulunran** [comment]
We are also having error on installation  on laravel 9 
plz guide me kindly

**Comment by abubakar0101** [comment]
I'm also facing this kind of issue. my client want me to develop shopify app on laravel 9 but i'm getting error in osiset package installation in laravel 9

**Comment by moonpetaldev** [comment]
This is being resolved in PR #1100
